core/templates/services/date-time-format.service.ts

```diff
@@ -25,10 +25,18 @@ import moment from 'moment';
   providedIn: 'root'
 })
 export class DateTimeFormatService {
-  // Returns just the time if the local datetime representation has the
-  // same date as the current date. Otherwise, returns just the date if the
-  // local datetime representation has the same year as the current date.
-  // Otherwise, returns the full date (with the year abbreviated).
+/**
+  * This function returns just the time if the local datetime representation
+  * has the same date as the current date. Otherwise, returns just the date
+  * if the local datetime representation has the same year as the current
+  * date. Otherwise, returns the full date (with the year abbreviated).
+  * @param {number} millisSinceEpoch - milliseconds since Epoch
+  * @returns {string} - The time if the local datetime representation has the
```

Comment:
Ditto for this. Also, I would say adding the complete description once in the main function description. Here you could simply say returns a date or something.

We should not add the same description in the return as well as the function definiton.

Reply:
Done